<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>checkwindow</class>
 <widget class="QWidget" name="checkwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>check</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTableWidget" name="checkWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>100</horstretch>
       <verstretch>100</verstretch>
      </sizepolicy>
     </property>
     <property name="autoScrollMargin">
      <number>16</number>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>209</number>
     </attribute>
     <column>
      <property name="text">
       <string>商品</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>数量</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>进价</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>售价</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>tips：先点击&quot;确认定价&quot;，再点击&quot;发布&quot;~</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>售价上的数字代表原先的定价，若为0则表示还未定价</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="check_btn">
         <property name="text">
          <string>确认定价</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="check_push">
         <property name="text">
          <string>上架</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="check_backbtn">
         <property name="text">
          <string>返回</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>check_backbtn</sender>
   <signal>clicked()</signal>
   <receiver>checkwindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>575</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
